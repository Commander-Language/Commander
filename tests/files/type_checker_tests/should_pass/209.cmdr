//Test that modulo set operation type checks
x = 1.2;
x %= 2.3;
