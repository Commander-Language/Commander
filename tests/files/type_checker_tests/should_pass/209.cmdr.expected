(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode x) SET (FloatExprNode 1.200000 FLOAT) FLOAT))
	(ExprStmtNode (BinOpExprNode (VarLValueNode x FLOAT) MODULO_SET (FloatExprNode 2.300000 FLOAT) FLOAT)))
